_data/meltano/extractors/tap-gainsightpx/Widen.yml

```
@@ -0,0 +1,61 @@
+name: tap-gainsight
+description: User tracking in web apps
+label: GainsightPX
+namespace: tap_gainsightpx
+logo_url: /assets/logos/extractors/gainsightpx.png
+variant: widen
+pip_url: git+https://github.com/Widen/tap-gainsightpx.git
+repo: https://github.com/Widen/tap-gainsightpx
+capabilities:
+- catalog
+- state
+- discover
+- about
+- stream-maps
+- schema-flattening
+settings_group_validation:
+- api_url
+- client_id
+- client_secret
+- refresh_token
+- start_date
+settings:
+- name: api_url
+  label: API URI
+  description: The base url for GainsightPX service. See GainsightPX docs.
+  documentation: https://support.gainsight.com/PX/API_for_Developers/02Usage_of_Different_APIs/Work_with_the_Gainsight_PX_REST_API
+- name: api_key
+  label: API Key
+  kind: password
+  description: The api key to authenticate against the GainsightPX service.
+  documentation: https://support.gainsight.com/PX/API_for_Developers/02Usage_of_Different_APIs/Work_with_the_Gainsight_PX_REST_API
+- name: page_size
+  label: Page Size
+  description: The number of records to return from the API in single page.Default and max varies based on the endpoint
+- name: start_date
+  label: Start Date
+  kind: date_iso8601
+  description: The earliest record date to sync (inclusive '>='). ISO Format.
+- name: end_date
+  label: End Date
+  kind: date_iso8601
+  description: The latest record date to sync (inclusive '<='). ISO format.
+- name: stream_maps
+  label: Stream Maps
+  description: Config object for stream maps capability. For more information check out [Stream Maps](https://sdk.meltano.com/en/latest/stream_maps.html).
+- name: stream_map_config
+  label: Stream Map Config
+  description: User-defined config values to be used within map expressions.
+- name: flattening_enabled
+  label: Flattening Enabled
+  description: 'True' to enable schema flattening and automatically expand nested properties.
```

```suggestion
  description: "'True' to enable schema flattening and automatically expand nested properties."
```
@jlloyd-widen the single quotes were causing a yaml syntax error. Popping them in double quotes looks to fix it 👍 .